src/VisualStudio/Core/Def/Implementation/ProjectSystem/LinkedFileUtilities.cs

```diff
@@ -70,9 +70,7 @@ private IVsHierarchy GetSharedItemContextHierarchy(IVisualStudioHostDocument doc
                 return null;
             }
 
-            var sharedHierarchy = document.SharedHierarchy;
```

Comment:
@Pilchie by the way, even editor case, it is possible for LinedFileUtilties to answer no for shared file. not sure what would happen in those case.